From jejb@kernel.org  Thu Oct 23 13:02:01 2008
From: Oliver Neukum <oliver@neukum.org>
Date: Thu, 23 Oct 2008 17:35:11 GMT
Subject: USB: cdc-wdm: make module autoload work
To: jejb@kernel.org, stable@kernel.org
Message-ID: <200810231735.m9NHZBH3008400@hera.kernel.org>

From: Oliver Neukum <oliver@neukum.org>

commit aa5380b904e7f896db0931320160bdea93e41f6a upstream

this fixes an omission that led to no alias being computed for the
cdc-wdm module.

Signed-off-by: Oliver Neukum <oneukum@suse.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/class/cdc-wdm.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/usb/class/cdc-wdm.c
+++ b/drivers/usb/class/cdc-wdm.c
@@ -42,6 +42,8 @@ static struct usb_device_id wdm_ids[] = 
 	{ }
 };
 
+MODULE_DEVICE_TABLE (usb, wdm_ids);
+
 #define WDM_MINOR_BASE	176
 
 
